### PR TITLE
MC-12347: Added missing trial documentation and fixed the doc after s…

### PR DESCRIPTION
**Pull request by fyrnaga**
…coping of trial and trails settings

#### Changes made

- Added Trial missing documentation (missing quota)
- Added the scope in the trials settings and trials

#### Related PRs
- PR # https://github.com/cloudops/cloudmc-core/pull/978
- PR # https://github.com/cloudops/cloudmc-ui/pull/1065
- PR # https://github.com/cloudops/cloudmc-api-docs/pull/255

### CODE REVIEW

File: source/includes/administration/_trials.md [hunk 1]
```diff
@@ -197,19 +207,491 @@ Updates a specific trial's settings.
...
+```json
+{
+  "data": 10
+}
+```
+Indicates how many organizations can concurrently have an active trial with automatic approval.
+
+Query Params | &nbsp;
+---- | -----------
+`organization_id`<br/>*UUID* | Return only settings for the provided organization id. If not provided, it will default to the reseller configurtion matching the url.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_trials.md]: configuration

File: source/includes/administration/_trials.md [hunk 2]
```diff
@@ -197,19 +207,491 @@ Updates a specific trial's settings.
...
+`username`<br/>*string* | The username of the person that made the activity.
+`userOrganizationId`<br/>*UUID* | The organization id of the user that made the activity.
+`userOrganizationName`<br/>*string* | The organization name of the user that made the activity.
+`userFirstName`<br/>*string* | The last name of the person that made the activity.
+`userLastName`<br/>*string* | The last name of the person that made the activity.
+`userEmail`<br/>*string* | The email address of the person that made the activity.
+`createdDate`<br/>*string* | The creation date of the activity.
+`updatedDate`<br/>*string* | The update date of the activity.
+`requesterIp`<br/>*string* | The requester ip address
+`parentId`<br/>*UUID* | The organization object create in the system for this organization once the trial is approved.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_trials.md]: created?

Comment (fyrnaga) [reply source/includes/administration/_trials.md]: e9223000cc97e8783f837e5dce7003320b071ca3

File: source/includes/administration/_trials.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/administration/_trials.md]: e9223000cc97e8783f837e5dce7003320b071ca3

File: source/includes/administration/_trials.md [hunk 3]
```diff
@@ -197,19 +207,491 @@ Updates a specific trial's settings.
...
+`resellerOrganizationId`<br/>*UUID* | The reseller organization id that the trial belongs to.
+`status`<br/>*string* | The status of the trial. Possible values are:  SUBMITTED, PENDING, DENIED, ONGOING, EXPIRED, PURGED, CONVERTED.
+`firstName`<br/>*string* | The first name of the person that requested the trial.
+`lastName`<br/>*string* | The last name of the person that requested the trial.
+`email`<br/>*string* | The email address of the person that requested the trial.
+`organizationName`<br/>*string* | The name of the organization for which the requester is doing the trial for.
+`createdDate`<br/>*string* | The creation date of the trial.
+`expiryDate`<br/>*string* | The expiration date of the trial.
+`persistedOrganizationId`<br/>*UUID* | The id of the organization created in the system once the trial is approved.
+`organization`<br/>*Object* | The organization object create in the system for this organization once the trial is approved.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_trials.md]: created

File: source/includes/administration/_trials.md [hunk 4]
```diff
@@ -197,19 +207,491 @@ Updates a specific trial's settings.
...
+`status`<br/>*string* | The status of the trial. Possible values are:  SUBMITTED, PENDING, DENIED, ONGOING, EXPIRED, PURGED, CONVERTED.
+`firstName`<br/>*string* | The first name of the person that requested the trial.
+`lastName`<br/>*string* | The last name of the person that requested the trial.
+`email`<br/>*string* | The email address of the person that requested the trial.
+`phoneNumber`<br/>*string* | The phone number of the person that requested the trial.
+`organizationName`<br/>*string* | The name of the organization for which the requester is doing the trial for.
+`createdDate`<br/>*string* | The creation date of the trial.
+`expiryDate`<br/>*string* | The expiration date of the trial.
+`persistedOrganizationId`<br/>*UUID* | The id of the organization created in the system once the trial is approved.
+`organization`<br/>*Object* | The organization object create in the system for this organization once the trial is approved.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_trials.md]: created

File: source/includes/administration/_trials.md [hunk 5]
```diff
@@ -197,19 +207,491 @@ Updates a specific trial's settings.
...
+`username`<br/>*string* | The username associate to the creator of the trial.
```
Comment (GuillaumeVialle) [inline source/includes/administration/_trials.md]: associated

**Review by GuillaumeVialle — COMMENTED** [review]
Sorry I missed these ones the first time

File: source/includes/administration/_trials.md (hunk 5 above)
Comment (fyrnaga) [reply source/includes/administration/_trials.md]: b218b4ee8649a770ebce55f95df8ff615d34416f

File: source/includes/administration/_trials.md (hunk 4 above)
Comment (fyrnaga) [reply source/includes/administration/_trials.md]: b218b4ee8649a770ebce55f95df8ff615d34416f

File: source/includes/administration/_trials.md (hunk 3 above)
Comment (fyrnaga) [reply source/includes/administration/_trials.md]: b218b4ee8649a770ebce55f95df8ff615d34416f

**Review by GuillaumeVialle — APPROVED** [review]

**Review by evanluc — APPROVED** [review]

**Pull request merged**